code/__DEFINES/conflict.dm

```diff
@@ -36,6 +36,7 @@
 ///Can only hit people with criminal status
 #define AMMO_MP					(1<<21)
 #define AMMO_FLAME				(1<<22) // Handles sentry flamers glob
+#define AMMO_STRIKES_SURFACEONL	(1<<23) // If the projectile hits a dense turf it'll do on_hit_turf on the turf just in front of the turf instead of on the turf itself (This one does not work on mobs)
```

Comment:
```suggestion
#define AMMO_STRIKES_SURFACE_ONLY	(1<<23) // If the projectile hits a dense turf it'll do on_hit_turf on the turf just in front of the turf instead of on the turf itself (This one does not work on mobs)
```